### PR TITLE
Re-export setRTLTextPlugin plugin

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: docs/get-started/get-started.md [hunk 1]
```diff
@@ -55,6 +55,19 @@ with Browserify or - [css-loader](https://webpack.github.io/docs/stylesheets.htm
 import 'mapbox-gl/dist/mapbox-gl.css';
 ```
 
+## Enable Right-to-Left Language Support
+
+```js
+import {setRTLTextPlugin} from 'react-map-gl';
+
+setRTLTextPlugin('https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.0/mapbox-gl-rtl-text.js');
```
Comment (NameFILIP) [inline docs/get-started/get-started.md]: the latest is `v0.2.3`:
* https://www.npmjs.com/package/@mapbox/mapbox-gl-rtl-text
* https://docs.mapbox.com/mapbox-gl-js/example/mapbox-gl-rtl-text/

**Comment by Pessimistress** [comment]
cc @jonsadka

**Comment by jonsadka** [comment]
> cc @jonsadka

Such great memory. Excited for this, thanks @Pessimistress

**Comment by NameFILIP** [comment]
@Pessimistress can we release a new version with this functionality?

**Comment by Pessimistress** [comment]
5.2.2

**Pull request merged**